README.md

```diff
@@ -18,6 +18,26 @@
   - [ ] Announcement talk at GitHub Universe
   - [ ] If not possible for Github Universe define alternative
 
+### Formally map 'Code as Data' enterprise needs
+- [ ] Do 8 user research interviews
+- [ ] Map formal user personas
+- [ ] Map key use cases
+  - [ ] Map features missing & priorities
+
```

Comment:
We might want to deliver a nice research output asset like an Experience Map https://medium.com/@wnialloconnor/how-to-build-an-experience-map-5e55b7ee4f32